shell/probe/raindrop.ps1

```diff
@@ -29,7 +30,6 @@ $Dat = ""
 
 while ($true) {
     try {
-        $VstDir = New-Item -ItemType Directory -Path ".vst" -Force
```

Comment:
Keep this, just pass the full path here, leave everything else the same

Reply:
ha, got this just as i did.. can you check if my change should work?

Reply:
Like use $Dir as the argument to this line, keep everything else the same

Reply:
did i do it...?

Reply:
No lol you need to capture this to a variable, just revert all your changes and add the one line at the top to pull the directory from the environment

Reply:
oh - i THINK i see... like this?

Reply:
i totally got it, didn't i?

Reply:
You did :)